packages/verification-vue-component/src/components/Verification/item/EventDetails.vue

```diff
@@ -1,8 +1,15 @@
 <template>
     <div class="event-details">
         <div v-for="detail in details" :key="detail.label" class="data-row">
-            <div v-if="detail.verifiable" :class="detail.verified === true ? 'verified' : 'unverified'">
-                <MDIcon :icon="mdiCheck"/>
+            <div v-if="detail.verifiable"
+                 :class="detail.verified === true ? 'verified' : 'unverified'"
+                 v-tooltip="{
+                    content: detail.verified === true ? $t('verification.result.meta.attributeVerified') : $t('verification.result.meta.attributeUnverified'),
+                    placement: 'right'
+                }"
```

Comment:
As a shortcut it should be possible to use it like that, also you need to use a different function for translation:
```suggestion
                 v-tooltip.right="detail.verified === true ? _$t('verification.result.meta.attributeVerified') : _$t('verification.result.meta.attributeUnverified')"
```